lib/fir/fir-basicblock.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
You probably don't need to prefix the names of the files in `fir/` with `fir-`.

Reply:
True. I borrowed from Swift.